frontend/src/metabase/query_builder/components/SavedQuestionIntroModal.tsx

```diff
@@ -60,14 +60,23 @@ export const SavedQuestionIntroModal = ({
       <Modal.Overlay />
       <Modal.Content p="xl" ta="center">
         <Modal.Header maw={contentWidth} mx="auto" my="md">
-          <Stack align="center">
+          <Stack align="center" justify="center" style={{ width: "100%" }}>
```

Comment:
we try to avoid using the `style` prop wherever possible. this should probably just be `<Center w="100%">`

https://mantine.dev/core/center/